packages/cozy-sharing/src/components/__snapshots__/ContactSuggestion.spec.jsx.snap

```diff
@@ -290,7 +290,7 @@ exports[`ContactSuggestion component should display contact suggestion for a gro
         style={
           Object {
             "--circleSize": "32px",
-            "backgroundColor": "#7F6BEE",
+            "backgroundColor": "var(--purpley)",
```

Comment:
Lots of non-semantic colors in those test components (it is their only use here).